src/list.js

```diff
@@ -106,15 +106,14 @@ List all the installed packages and also the packages bundled with Atom.\
     listPackages(directoryPath, options) {
       const packages = [];
       for (let child of Array.from(fs.list(directoryPath))) {
-        var manifestPath;
         if (!fs.isDirectorySync(path.join(directoryPath, child))) { continue; }
         if (child.match(/^\./)) { continue; }
         if (!options.argv.links) {
           if (fs.isSymbolicLinkSync(path.join(directoryPath, child))) { continue; }
         }
 
-        let manifest = null;
```

Comment:
Might have been deleted by accident, `manifest` is checked later in the function.

We could probably refactor so it's not checked later, if we want, but for now we can restore this line `let manifest = null;`

Reply:
Thanks for calling this one out, restored that line, not sure what went wrong there